docs/nodes/blobstream-intro.md

```diff
@@ -0,0 +1,110 @@
+
+## Overview
+
+Blobstream,
+consists of two components: an [Orchestrator](../blobstream-orchestrator)
+and a [Relayer](../blobstream-relayer).
+
+In the following diagram, we show how a layer 2 would post the data to
+Celestia. This will later be attested to by the Celestia validator set, and
+eventually posted to the target EVM chain (in this case, Ethereum). Then,
+the layer 2, or any party, will be able to verify the attestations, i.e. valsets
+and data commitments, directly on the EVM chain on the Blobstream smart contract. You can
+reference
+[the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
+
+![Blobstream-Architecture](../img/blobstream/Blobstream.png)
+
+The specification of Blobstream `Valset`s, which track the Celestia validator set
+changes, can be found in [ADR 002](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-002-qgb-valset.md).
+
+Blobstream data commitments, which represent commitments over sets of blocks
+defined by a data commitment window, are
+discussed more in-depth in
+[ADR 003](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-003-qgb-data-commitments.md).
+
+The Orchestrator is part of the validator setup and works as follows:
+
+- Celestia App: creates an attestation on the state machine level that needs to
+  be signed
+- The Orchestrator: queries the attestation, signs it, then submits the signature
+  back to Celestia App
+
+The diagram below goes over this process.
+
+![Blobstream-Orchestrator](../img/blobstream/blobstream-orchestrator.png)
+
+The Relayer deploys the Blobstream smart contract first to the EVM L1 chain (if it is
+not deployed before), and then relays the attestations from Celestia App to the
+EVM L1 Chain.
+
+The diagram below goes over this process.
+
+![Blobstream-Relayer](../img/blobstream/blobstream-relayer.png)
+
+You can learn more about the mechanics behind the Relayer in
+[ADR 004](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-004-qgb-relayer-security.md).
+
+## Blobstream and data availability committees (DACs)
+
+### Decentralization and security
+
+Blobstream is built on Celestia, which uses a CometBFT-based proof-of-stake
+system. An incorrect data availability attestation in this system can be
+penalized, ensuring validators act in good faith. In contrast, data availability
+committees (DACs), are typically centralized or semi-centralized, relying on
+a specific set of entities or individuals to vouch for data avaiability.
+
+### Mechanism of verification
+
+Blobstream uses data availability attestations, which are Merkle roots of the
+L2 data, to confirm that the necessary data is present on Celestia. When
+Ethereum's L2 contract updates its state, it checks with Blobstream to confirm
+that data's presence on Celestia. In contrast, a DAC would rely on attestations
```

Comment:
> it checks with Blobstream to confirm that data's presence on Celestia

How this? I am not aware of this happening or I don't get this part

Reply:
reworked in https://github.com/celestiaorg/docs/pull/1115/commits/ba3b63238fa88379334c509b748d14ec8ad42f91